### PR TITLE
DAM-2124 Adding Bacon

### PR DESCRIPTION
**Pull request by svjoshua**
I am adding bacon to this issue to test a new workflow

### CODE REVIEW

**Comment by svjoshua** [comment]
I am adding a comment to the thread now

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # workflow-testing
 
 I am modifying the README
+I am adding bacon to test a new workflow
```
Comment (evenmoor) [inline README.md:4]: I think there is a misspelling here.

Comment (evenmoor) [reply README.md:4]: Another comment

**Comment by svjoshua** [comment]
Adding a comment in the main body

**Comment by svjoshua** [comment]
Adding another comment in the main body

File: README.md (hunk 1 above)
Comment (evenmoor) [reply README.md:4]: Testing continued!

Comment (evenmoor) [reply README.md:4]: Can you see this in Jira?

Comment (evenmoor) [reply README.md:4]: How about this?

Comment (evenmoor) [reply README.md:4]: And this?

Comment (evenmoor) [reply README.md:4]: This has to be a review comment right?

**Review by evenmoor — CHANGES_REQUESTED** [review]
Please make an update

**Review by evenmoor — APPROVED** [review]
This is where I should write up some QA notes

1. Step 1
2. Step 2
3. Step 3
4. Step 4

**Pull request closed without merging**